migration/mule/jms-message-rollback-and-redelivery/README.md

```diff
@@ -0,0 +1,67 @@
+# JMS message rollback and redelivery
+
+This example shows how to implement jms rollback and redelivery within the WSO2 Integration Studio. While when we passing the jms messages inside the WSO2 EI, following failures could be happening.
+* Failures during message mediation process.
+* Message sending failure,due to endpoint is unavailability.
+
+The JMS message passing flow in this example is configured with the ActiveMQ MOM. The requirement is if any of the above failures happen, the message should not process further and send to end point, rather it should roll back. After a specific number of unsuccessful failures happen the message should send to ActiveMQ topic.
```

Comment:
What is meant by `send to end point` here? Shall we say `message should not be acknowledged, it should be rollbacked`.

IMO, `After a specific number of unsuccessful failures happen the message should send to ActiveMQ topic.` should come under `Example Use Case` section, not at the introduction.